app/unitPages.rb

```diff
@@ -582,6 +582,11 @@ def getIssueNumberingTitle(unit_id, volume, issue)
   return attrs['numbering'], attrs['title'], attrs['show_pub_dates']
 end
 
+
```

Comment:
If this empty method is still present in the deployed code, it should be removed.